code/modules/surgery/organs/vocal_cords.dm

```diff
@@ -119,12 +119,19 @@
 ///////////VOICE OF GOD///////////////
 //////////////////////////////////////
 
-/proc/voice_of_god(message, mob/living/user, list/span_list, base_multiplier = 1, include_speaker = FALSE, message_admins = TRUE)
+/proc/voice_of_god(message, mob/living/user, list/span_list, base_multiplier = 1, include_speaker = FALSE, message_admins = TRUE, datum/language/_language = null)
 	var/cooldown = 0
+	message = uppertext(message)
 
-	if(!user || !user.can_speak() || user.stat)
+	if(!user || user.stat)
 		return 0 //no cooldown
 
+	if(!_language)
+		_language = user.get_default_language()
+
+	if(!user.say(message, spans = span_list, sanitize = FALSE, language = _language))
```

Comment:
Why sanitize is false ?

Reply:
It's been a few months since i made this PR, but i think that it should already be sanitized by then, either by stripped input or by the original say proc when activated via .x, and sanitizing it again apparently breaks symbols like ' and so on.

Reply:
That might be true for using the key but now you're passing it directly from colossus say. I mean the whole thing is adminabuse territory anyway but i'd rather not have them screaming images.

Reply:
Oh yeah, you're right. Still, double-sanitizing causes issues as i said, so i might sanitize manually on the colossus say, or something like that.